cmd/edv-rest/startcmd/start.go

```diff
@@ -83,17 +94,27 @@ type edvParameters struct {
 	databaseURL    string
 	databasePrefix string
 	logLevel       string
+	tlsConfig      *tlsConfig
+}
+
+type tlsConfig struct {
+	certFile string
+	keyFile  string
 }
 
 type server interface {
-	ListenAndServe(host string, router http.Handler) error
+	ListenAndServe(host, certFile, keyFile string, router http.Handler) error
```

Comment:
it should be probably `ListenAndServeTLS`

Reply:
One problem... the ListenAndServe implementation can also call ListenAndServe (without TLS) if the provided certFile and keyFile are missing. My concern is that the name will be misleading in that case...and so "ListenAndServe" technically covers both cases. Also, I pulled this from other TrustBloc projects where they do the same thing.

What do you think? @llorllale @aholovko ?

Reply:
Disregard my last message - I'm going to make it TLS only per the suggestion below, so I will update the interface and method names.